src/batches/setBatchGZip.m

```diff
@@ -0,0 +1,22 @@
+% (C) Copyright 2020 CPP BIDS SPM-pipeline developers
+
+function matlabbatch = setBatchGZip(unzippedNiifiles, keepUnzippedNii)
+  %
+  % Set the batch for GZip the 4D volumes
+  %
+  % USAGE:: 
+  %
+  %   matlabbatch = setGZip(unzippedNiifiles, deleteUnzippedNii)
+  %
+  % :param unzippedNiifiles: List of volumes to be gzipped
+  % :type unzippedNiifiles: array
+  % :param keepUnzippedNii: Obligatory argument. Boolean to decide to delete the unzipped files
```

Comment:
```suggestion
  % :param keepUnzippedNii: Boolean to decide to delete the unzipped files
```

unless the argument is in sqare brackets in the USAGE section then it is obligatory